src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -1792,6 +1793,9 @@ private static MeshGeometry3D DrawTestMesh()
 
         internal void UpdateNearClipPlane()
         {
+
+            if (camera == null) return;
```

Comment:
I think it is safe to check the compare (camera position) here. I am not sure though.